### DISCUSSION TITLE
HTTP Listener first time using

### DISCUSSION THREAD

**Discussion by rutledj** [nextgenhealthcare/connect#4756] [Q&A] [answered]
I have a supplied channel from a vendor that uses an HTTP listener.  I'm unfamiliar with these and wasn't sure what the Base Context Path value should be or is used for. As supplied it is blank. Does it require a value?

**Comment by pacmano1** [comment, marked as answer]
Adding a value in there makes `https://somehost/` -> `https://somehost/<the context path>`

Depends on your use case.   In general for standing up inbould APIs I do use one.
